### PR TITLE
Fixing #101

### PR DESCRIPTION
**Pull request by StevenClontz**
Hi @tdegeorge - I think you forgot to make a clean branch off of main for this last pull request, so I needed to tweak a few things to get your contributions back into the main branch.

I'm also about to add a couple fixes for future reference. Once I'm done I'll have @siwelwerd review and merge.

### CODE REVIEW

File: source/02-FN/01.ptx [hunk 1]
```diff
@@ -71,7 +71,7 @@
     <definition xml:id="def-function">
       <statement>
         <p>
-          A <term>function</term> is a relation where every input (or <m>x</m>-value) is mapped onto <term>**exactly one**</term> output (or <m>y</m>-value).</p>
+          A <term>function</term> is a relation where every input (or <m>x</m>-value) is mapped onto <em>exactly one</em> output (or <m>y</m>-value).</p>
```
Comment (StevenClontz) [inline source/02-FN/01.ptx:74]: Only use `<term>` when introducing new terminology, and use `<em>` when you want to emphasize something. They look the same in HTML, but make a big difference to the (say) braille reader.

File: source/02-FN/01.ptx [hunk 2]
```diff
@@ -364,7 +364,7 @@
 </activity>
 
 <remark>
-  Notice in <xref ref="FN1-determine-if-function-from-equation"/> that when you are given an equation with a <m>y^2</m> term, the equation is not a function.
+  Notice that <xref ref="FN1-determine-if-function-from-equation"/> shows that equations with a <m>y^2</m> term generally do not define functions.
```
Comment (StevenClontz) [inline source/02-FN/01.ptx:367]: Technically $y^2=0$ defines a function, so this is a little more careful.

**Review by siwelwerd — APPROVED** [review]

File: source/02-FN/01.ptx (hunk 2 above)
Comment (siwelwerd) [reply source/02-FN/01.ptx:367]: Also some fun shapes like $y^3=x^2$

Comment (StevenClontz) [reply source/02-FN/01.ptx:367]: ? That's a function but it doesn't have a $y^2$. And $y^2=x^3$ isn't a function.

Comment (siwelwerd) [reply source/02-FN/01.ptx:367]: Students often focus more on the "any higher power of $y$" than the $y^2$ in particular.

Comment (StevenClontz) [reply source/02-FN/01.ptx:367]: Point taken - makes me wonder if this remark is overall helpful or has potential to unintentionally mislead...

Comment (siwelwerd) [reply source/02-FN/01.ptx:367]: I think if $f$ is any bijective function, and $g$ is just a function, then $f(y)=g(x)$ is a function (as then $y=f^{-1}(g(x))$).  Even if you can't explicitly solve for $f^{-1}$.

**Pull request merged**